template/{{.input_root_dir}}/cicd_params.json.tmpl

```diff
@@ -0,0 +1,8 @@
+{
+  "input_root_dir": "{{ .input_root_dir }}",
+  "input_cloud": "{{ .input_cloud }}",
+  "input_cicd_platform": "{{ .input_cicd_platform }}",
```

Comment:
We let user select the input_cloud during both the CICD/root_dir creation and the project creation.
- CICD: the cloud is used to in the cicd creation workflow and some other places
- project: project initialization uses the cicd platform to decide node type, workspace, etc.

We need to add a validation to make sure the could is the same between the parent and the project during the CICD creation workflow.

Reply:
The easiest way is to add a new input vairable `input_project_cicd_platform` to `fs_param.json`(rename it to `project_param.json`) and check if the two input variables are the same.

Reply:
You mean `input_cloud` right?

Reply:
Added a comparison check for `input_cloud`.